src/bp/ui-studio/src/web/components/Shared/Interface/index.tsx

```diff
@@ -173,9 +173,10 @@ export const InfoTooltip: FC<InfoTooltipProps> = props => (
 )
 
 const SectionAction = (action: SectionAction, idx: number) => {
+  const toolTipPos = action.tooltipPosition ? action.tooltipPosition : Position.RIGHT
   if (action.items) {
     return (
-      <Tooltip key={idx} disabled={!action.tooltip} content={action.tooltip} position={Position.RIGHT}>
+      <Tooltip key={idx} disabled={!action.tooltip} content={action.tooltip} position={toolTipPos}>
```

Comment:
I'm wondering of the usefulness of this parameter, since we want to have a similar interface. Why isn't it good on the right ?